test/library/packages/LinearAlgebra/correctness/testMatrix-error.chpl

```diff
@@ -0,0 +1,11 @@
+use LinearAlgebra;
```

Comment:
Great test! Would you mind putting this in `correctness/no-dependencies` since it has no LAPACK/BLAS dependencies?

Reply:
Thank you! The test should be moved to the proper location now.